fastats/linear_algebra/__init__.py

```diff
@@ -28,4 +29,5 @@
     'standard_error',
     'mean_standard_error_residuals',
     't_statistic',
+    'drop_missing'
```

Comment:
I like having commas at the end of lists, dicts etc spanning multiple lines - this shortens the diff

Reply:
Thanks for the review - sure thing, sounds good